rest/model/build.go

```diff
@@ -132,8 +132,6 @@ func (apiBuild *APIBuild) BuildFromService(v build.Build, pp *model.ParserProjec
 		origin = commitOrigin
 	case evergreen.GithubPRRequester:
 		origin = patchOrigin
-	case evergreen.MergeTestRequester:
```

Comment:
Not exactly related to the deletion (which looks fine), but is it intentional that this switch doesn't have a case for GitHub merge queues?